src/components/BaseText/BaseText.tsx

```diff
@@ -0,0 +1,14 @@
+import './BaseText.css';
+
+interface BaseTextProps {
+  children: React.ReactNode;
+  size?: 'xl' | 'l' | 'm' | 's';
```

Comment:
Actualizar los posibles valores por una version de 2 caracteres, y a su vez tambien realizar el cambio en el archivo css para que haga match con estos nuevos nombres

```suggestion
  size?: 'xl' | 'lg' | 'md' | 'sm';
```